explorer/mquery.go

```diff
@@ -280,6 +280,10 @@ func (m *Mquery) refreshChecksumAndType(queries map[string]*Mquery, props map[st
 
 // RefreshAsFilter filters treats this query as an asset filter and sets its Mrn, Title, and Checksum
 func (m *Mquery) RefreshAsFilter(mrn string, schema llx.Schema) (*llx.CodeBundle, error) {
+	if m.Title == "" {
```

Comment:
is there any reason why we fall back to the query if title is empty? is title being empty bad?

Reply:
I also do not understand why this is required

Reply:
the title is part of the fields used to calculate the checksum of the query

Reply:
I also do not understand. I can remove it if everyone agrees

Reply:
@imilchev the checksumming doesn't care if the title is empty. What would be important is its consistently empty.

The one way it may break is our checksum is something like:
```
c = c.Add(A)
c = c.Add(B)
c = c.Add(C)
```
and i think it does it a byte at a time, so the checksums for `(A, B, C) = ("foo", "bar", "")` would be the same as `(A, B, C) = ("foo", "", "bar")`. But I didn't look deep into it and I may be misremembering how it actually works

Reply:
I suspect the query-as-title fallback has been used to make printing on CLI pretty, so we should probably move that logic there

Reply:
do we print asset filters?

Reply:
nope, i keep forgetting this change is only for asset filters